### ISSUE TITLE
Sort columns in Filtering log.

### ISSUE THREAD

**Issue by XX-J** [AdguardTeam/AdguardBrowserExtension#1127]
&nbsp;1. Сортировка столбцов по нажатию на заголовок (восходящая, нисходящая) с запоминанием между браузерными сессиями.
&nbsp;~2. Перемещение разделителя столбцов с запоминанием между браузерными сессиями.~

![2022 07 17-2](https://user-images.githubusercontent.com/17948007/179390637-119d3811-cfc9-4e13-a6ee-fdbab225deb6.png)
<br>

----------
 AdGuard v4.0.204 &nbsp; | &nbsp; FireFox v78.13 ESR &nbsp; | &nbsp; Win7SP1

**Comment by XX-J** [comment]
Можно как временную замену сортировки (если её сложно реализовать) или как дополнение к ней раскрасить текст типов подгружаемых данных. При быстром пролистывании длинных списков это-бы помогло боковым зрением быстрее определять к какому типу данных относится читаемая в данный момент строчка.

<details><summary>Пример раскраски (цвет текста должен соответствовать цвету его обвода):</summary>

![2018 10 13-2](https://user-images.githubusercontent.com/17948007/47920831-1873d200-dec4-11e8-9a86-eb45c4b47e75.png)
</details>

**Comment by Alex-302** [comment]
Да, цветовая дифференциация ~штанов~ контент-тайпов была бы удобна.
